fe1-web/network/__tests__/MessageApi.test.ts

```diff
@@ -4,12 +4,18 @@ import '__tests__/utils/matchers';
 import 'store/Storage';
```

Comment:
not your fault, but this file is really really ugly.
Don't feel forced to comply with its current logic, you can start breaking it apart if you think it'll make your life easier.

Reply:
Yes, it might be a really good idea to refactor it if we have the time to do it, but it will probably be a lot of work. For now, I think it's usable but not practical at all.

Reply:
I think It might be a really good idea to refactor it if we have the time to do it.
You're right, I moved my code in another file so that this file doesn't get any longer.